ch_tools/chadmin/internal/dictionary.py

```diff
@@ -1,8 +1,12 @@
+from pathlib import Path
 from typing import Any, Optional
 
 from click import Context
 
 from ch_tools.chadmin.internal.utils import execute_query
+from ch_tools.common import logging
+from ch_tools.common.clickhouse.config.path import CLICKHOUSE_SERVER_CONFIG_PATH
+from ch_tools.common.clickhouse.config.utils import _load_config
```

Comment:
Underscore prefix in method name means that it is a private method and should not be used outside it's module. Public `load_config` method can be used here instead